### PR TITLE
first commit home work 13

### PR DESCRIPTION
**Pull request by Oleksii1985**

### CODE REVIEW

File: lesson_6/usersAccounts.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    "balance": "$1,790.56",
+    "name": "Suzette Lewis",
+    "gender": "female",
+    "phone": "+1 (837) 586-3283",
+    "address": "314 Dunne Place, Bawcomville, Guam, 9053"
+  }
+]
+
+// balance with more 2000 dollars
+let balanceMore = users.filter(user => user.balance > '$2,000.00');
```
Comment (web-scavenger) [inline lesson_6/usersAccounts.js]: you sort strings, not numbers. It could make bugs.

File: lesson_6/usersAccounts.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+console.log(balanceMore);
+
+// phone numbers who has more 2000 dollars balance
+let telNumbers = balanceMore.map(user => user.phone + ' ');
```
Comment (web-scavenger) [inline lesson_6/usersAccounts.js]: why do you need `' '`

Comment (Oleksii1985) [reply lesson_6/usersAccounts.js]: done

File: lesson_6/usersAccounts.js (hunk 1 above)
Comment (Oleksii1985) [reply lesson_6/usersAccounts.js]: done

**Pull request merged**